surveyPageOne.php

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang ="en" >
+    <head>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <title> Survey </title>
+        <meta charset="UTF-8">
+        <meta name="description" content="Survey Page for Cs464 Experiment">
+        <meta name="keywords" content="cs464, CSU, CSS, HTML">
+    </head>
+    <body>
+        <br><br><br><br>
+        <div class="question-container">
+            <h4><center>Question 5</center><h4>
+            <h5><center>What month were you born in?</center><h5>
+                <div>
+                    <input type="radio" value="Under 18" id="Q1A1" name="Q1"/>
```

Comment:
The "value" has to be changed to the value of the label - this is what is sent to the "DB" as their answer rather than the label.

Reply:
(This would have to be changed in each question if the value doesn't match the label

Reply:
id properties have to also be unique

Reply:
(For each php page - not in the entire project

Reply:
name in each question should be changed to question name (i.e. for this one -> name = "Q15")

Sorry for bombarding - I didn't mean to separate all of these comments lol

Reply:
- actually technically speaking I don't think name has to change so you can disregard that 🥲

Reply:
I think I fixed it all!